mothertongues/config/models.py

```diff
@@ -96,10 +96,10 @@ def convert_sub_costs(cls, values):
             if v_path.suffix == ".json":
                 with open(v_path, encoding="utf8") as f:
                     data = json.load(f)
-            if v_path.suffix == ".xlsx":
+            elif v_path.suffix == ".xlsx":
```

Comment:
@roedoejet, can you espeically look at this part? I don't think it's a breaking change, but wanted your eyes on this...

(without it, any filetype goes through without warning, just doesn't load any substitutions, which felt misleading)